test/test_linalg.py

```diff
@@ -1335,6 +1335,9 @@ def get_value_type(dtype):
         for keepdim in [True, False]:
             for input_size, ord_settings in test_cases:
                 for ord in ord_settings:
+                    if not torch._C.has_lapack and ord in [2, -2, 'nuc']:
```

Comment:
What's going on here and does this LAPACK check apply to every instantiation of this test, like including the CUDA instantiation?

Reply:
The condition is incomplete, yes the skip should be applied only for CPU device. I changed that.